content/work/quarqnet.rst

```diff
@@ -48,3 +100,296 @@ summary: |
+
+*As an athlete, I can view my activity (map & data).*
+
+Communication
+~~~~~~~~~~~~~
+
+Before diving into process details,
+a word about communication.
+At the beginning of each project,
+we establish a communication system
+for designers, developers, and clients -
+a shared Slack channel
+for daily communication and
+a schedule for weekly video meetings.
+User stories are prioritized in Pivotal Tracker
+and contain scope goals and a task list.
+To establish color patterns for QuarqNet,
+OddBird designers started with
+the existing brand colors
+and selected related user interface colors.
+These colors were communicated via Pivotal Tracker
+and coded as Sass Maps
+to establish front-end development patterns
+as well as an automated "living" `style guide`_.
+
+.. _style guide: http://quarqnet.com/styleguide/config-colors.html
+
+.. figure:: /static/images/work/quarqnet/communication.jpg
+   :class: extend-large
+   :alt: user story in Pivotal Tracker and Sass color maps
+
+Initial Sketches
+~~~~~~~~~~~~~~~~
+
+.. image:: /static/images/work/quarqnet/activity.jpg
+   :class: extend-left img-shadow
+   :alt: initial sketch of activity map and data
+
+Because of our focus on users,
+OddBird's development process
+for each user story
+begins with rough sketches
+showing the flow of the user
+through the content.
+The athletes using QuarqNet
+needed easy access to the map of their route
+as well as the activity data.
+This initial design proposal
+displays the map and activity details together
+in a single view.
+No clicking necessary
+to access the most useful information
+on both mobile and larger screens.
+
+Hi-Fi Mockups
+~~~~~~~~~~~~~
+
+After the design direction
+for a user story is approved,
+we create static or interactive mockups
+often using Adobe Xd.
+These mockups allow us
+to experiment with brand implementation
+as well as clarify user flow and interaction -
+identifying sticking points
+and iterating quickly.
+In conversation with Quarq,
+it became clear that activity data needed a new layout
+to improve scannability.
+We don't linger too long on this step
+or attempt pixel perfection
+as new UX problems nearly always present themselves
+during implementation.
+
+.. image:: /static/images/work/quarqnet/mockup1.jpg
+   :class: extend-large
+   :alt: initial mockups of activity map and data
+
+Implementation
+~~~~~~~~~~~~~~
+
+Additional UX design iteration
+happens during implementation
+as we begin to interact with the new features
+in the context of the browser.
+Whether working with our own internal developers
+or partnering with external developers,
+we maintain a tight a feedback loop
+between designers and developers
+as well as the clients at this stage.
+We have frequent video calls
+to demo, discuss, pair, and re-design features.
+For this QuarqNet feature,
+we improved the dotted line, font size, and spacing
+of the activity data
+within the browser.
+
+.. image:: /static/images/work/quarqnet/implementation1.jpg
```

Comment:
Yeah, that's the part you're talking about here, I think. The rest just looks a bit odd out of context.